tcf_website/templates/reviews/new_review.html

```diff
@@ -26,32 +28,44 @@ <h1 class="h2 mt-2 mb-4">Write a review</h1>
 {% block js %}
 <script type="text/javascript" src="{% static 'reviews/new_review.js' %}"></script>
 <script type="text/javascript" src="{% static 'reviews/sync_review_sliders.js' %}"></script>
+<script src="jquery-3.5.1.min.js"></script>
```

Comment:
We already have jquery linked in `index.html` and it gets inherited, so this isn't necessary.